### ISSUE TITLE
0x7f6de2ffd700 Error parsing content

### ISSUE THREAD

**Issue by daymanbpi** [macr0dev/Audiobooks.bundle#45]
Having an issue with the latest public release of plex. Getting the below errors when i try to match. I installed the plugin clean. Not sure the issue.

Plex Version 1.18.0.1846

0x7f6de2ffd700 Error parsing content
Exception caught while attempting to match metadata on item 135442: Error parsing file
XML was (0 bytes):

**Comment by daymanbpi** [comment]
Still getting the same error after upgrading to the latest public release Version 1.18.0.1913.

**Comment by ibeadam** [comment]
I get the same error, was working fine and I've not update Plex server (version  1.18.0.1846).

[0x14693ece5700] Error — Unknown metadata type: folder
[0x1468f6f37700] Error — Unknown metadata type: folder
[0x14693ece5700] Error — Error parsing content.
[0x14693ece5700] Error — Exception caught while attempting to match metadata on item 196542: Error parsing file
[0x14693ece5700] Error — XML was (0 bytes):
[0x1468e5cae700] Warning — No matches for search.

**Comment by BluebonnetSK** [comment]
I am having the same issue.

**Comment by macr0dev** [comment]
Let's get some more details.  My server appears to still be working since I just added a new book without error.

Are you using audible.com or one of the foreign sites?

**Comment by daymanbpi** [comment]
I have been using the default audible.com, but i get the same error on all the sites. 
I am on Linux - Wine, if that matters.

**Comment by macr0dev** [comment]
Are you telling me that you're running the Windows version of Plex on Wine, on Linux?  I'm not sure I get what you're telling me, but that's OK - it shouldn't really matter (I don't think) 

Drop me a book name or two that you're working with.

**Comment by daymanbpi** [comment]
No i was just saying i am running Plex on Linux OS.
It doesn't matter what the book title is. I get the error. If i try to search (fix match) on a previously matched book that already has meta-data, i get the error.

**Comment by macr0dev** [comment]
OK.  So I've managed to get myself an error on my end, but not quite the same.  You're right that the book doesn't matter - but I'm not getting any error on initial matching or refreshing meta data.  Only when I do a call for a manual match.  I wonder what Plex broke with their update....

**Comment by macr0dev** [comment]
So it looks like they've changed a few things that have effectively broken how the manual search looks.  They're handing off some variables differently that are causing some problems.  I'm going to do some digging but it's going to be a little trial and error on my side since I'm sure they aren't publishing their changes as they decrease support for plugins, etc....

**Comment by daymanbpi** [comment]
I will be the first one to say i appreciate your work on this issue. I love your plugin. Hopefully you can get it sorted. Thanks man.

**Comment by macr0dev** [comment]
Thanks.  It's much appreciated.  I'm honestly hoping they crank out another update in the next day or two because  _something_ is being goofy. 

I've got it running smoothly, except that the manually entered search terms aren't being on the fix match screen isn't being handed off properly.  So it will run and automatch, but if you need to fix it there's no way to change what it's searching for.

**Comment by ibeadam** [comment]
Unfortunately been busy work-wise so not been able to contribute anything of use. However, can I be the second to say how much I appreciate your work!

**Comment by kravick** [comment]
I should've spoken up sooner. Also really appreciate your work on the plugin. It's just too bad everytime Plex sends an update, i have delete and recopy the bundle to activate it a second time. But that's no fault of yours. Thanks for all the hard work.

**Comment by daymanbpi** [comment]
https://www.reddit.com/r/PleX/comments/dj4x95/new_beta_pms_version_available_118119730f4abfbcc/
ITEMS FIXED:
(Metadata) Improved reliability of music lookups

Not sure if this affects anything at all, Just thought i would post, haven't updated plex yet..

**Comment by macr0dev** [comment]
I found some stuff in the official Plex forums that indicates they're revamping some of the music stuff.  So that tracks with what I've seen.  I've started a thread in Plex's developer's forum and I was shocked to see the CTO reply. 

They've definitely changed a few things.  Hopefully they can tell me what variable I need to be referencing to get access to the manual search entry.  That's the final piece of the puzzle at the moment.

**Comment by macr0dev** [comment]
Good news everyone.  I've been told by the good people at Plex that they've discovered what was broken and are going to fix it in an upcoming release.  Here's a link to the thread if you're interested.  I'll leave this issue open until we've fully resolved the issue.  

https://forums.plex.tv/t/music-library-changes-in-recent-updates/477253/5

**Comment by alvarow** [comment]
Hi I ended up here trying to figure out why the agent fails to search .m4b ... I am still unable to figure that out, I'd like to report that if I convert the files to .m4a, then it has no problems!

`ffmpeg -i abook.m4b -map 0 -c copy abook.m4a`

Then the resulting book gets matched. I it is not a simple extension change... here's some output of mediainfo from a file that failed:

```
Format profile                           : Apple audio with iTunes position
Codec ID                                 : M4B  (M4B /mp42/isom)
```
then after telling ffmpeg to change container:
```
Format profile                           : Apple audio with iTunes info
Codec ID                                 : M4A  (isom/iso2)
```

I understand this is unrelated to the PMS changes that were fixed, just adding info is someone ends up here like I did.
